internal/runbits/org/org.go

```diff
@@ -0,0 +1,63 @@
+package org
```

Comment:
This comes from the init runner, which selects a default org if one wasn't specified. This code is not new.